README.md

```diff
@@ -0,0 +1,28 @@
+MVICore21
+=
+
+[![CircleCI](https://circleci.com/gh/21Buttons/MVICore21.svg?style=shield)](https://circleci.com/gh/21Buttons/MVICore21) 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
+
+## Description
+This library is based on [MVICore](https://github.com/badoo/MVICore) library from Badoo. 
+Its purpose is to create a lightweight framework to implement a MVI architecture in Android.
+Refer to [MVICore documentation](https://badoo.github.io/MVICore/) and [this post](https://badootech.badoo.com/a-modern-kotlin-based-mvi-architecture-9924e08efab1)
+to have a better insight on the usage of this framework.
+
+## Main differences with MVICore
+- **No Reducer class:** In MVICore21, the **Effect** class knows how to 'Reduce' itself, thus creating
+a new state
+- **No work done when creating the feature:** MVICore had a problem that was that just after creating an instance,
```

Comment:
I think they were not considering that a problem, so maybe we could simply say their approach didn't fit our needs.

Reply:
Yes, it was a problem for us, not them so I'll rephrase it